src/store/page/page-element.ts

```diff
@@ -157,69 +183,67 @@ export class PageElement {
 	 * Adds a child element to is element (and removes it from any other parent).
 	 * @param child The child element to add.
 	 * @param index The 0-based new position within the children.
-	 * Leaving out the position adds it at the end of the list.
```

Comment:
Why did you drop this note? It is still true, isn't it?

Reply:
whoops. got dropped by copy & paste i guess